build_projects/shared-build-targets-utils/Utils/Monikers.cs

```diff
@@ -11,13 +11,21 @@ public class Monikers
     {
         public const string SharedFrameworkName = "Microsoft.NETCore.App";
         public const string CLISdkBrandName = "Microsoft .NET Core 1.0.0 - SDK Preview 2";
+        public const string RuntimeName = "Microsoft .NET Core";
 
         private static string GetBrandName(BuildTargetContext c, string suffix)
         {
             var buildVersion = c.BuildContext.Get<BuildVersion>("BuildVersion");
-            return String.Format("Microsoft .NET Core {0}.{1}.{2} - {3}", buildVersion.Major, 
-            buildVersion.Minor, buildVersion.Patch, suffix);
+            var releaseBrandingSuffix = c.BuildContext.Get<string>("ReleaseBrandingSuffix")?.Trim();
+
+            string brandName = string.IsNullOrEmpty(releaseBrandingSuffix) ?
+                               //RTM branding
+                               $"{RuntimeName} {buildVersion.Major}.{buildVersion.Minor}.{buildVersion.Patch} {suffix}"
```

Comment:
Since RuntimeName/Major/Minor/Patch are the same for both RTM and Pre-release, I would suggest factoring these out to be a different string which is then used in the brandName initialization.